website/pages/en/_meta.js

```diff
@@ -5,10 +5,10 @@ export default {
   },
   about: '',
   network: 'The Graph Network',
-  sunrise: 'Sunrise Upgrade FAQ',
+  sunrise: '',
```

Comment:
I see you moved "Sunrise Upgrade FAQ" out of this file into `sunrise.mdx`'s `sidebarTitle` frontmatter, but why not do it for every page? Like "The Graph Network" just above?

Also, I guess the only purpose of these `_meta.js` files now is to set the order of the pages and add headings and separators?

Reply:
> I see you moved "Sunrise Upgrade FAQ" out of this file into sunrise.mdx's sidebarTitle frontmatter, but why not do it for every page? Like "The Graph Network" just above?

yep, I can do it in another PR

> Also, I guess the only purpose of these _meta.js files now is to set the order of the pages and add headings and separators?

exactly